src/FSLibrary/StellarCoreHTTP.fs

```diff
@@ -272,15 +286,7 @@ type MissionContext with
               spikeinterval = self.spikeInterval
               offset = 0
               maxfeerate = self.maxFeeRate
-              skiplowfeetxs = self.skipLowFeeTxs
```

Comment:
Why remove this? Aren't some missions using these defaults? (seems a bit easier to just use a pre-set rather than configure invoke host options manually, which is pretty involved)

Reply:
My other comment mostly explains this, but because these aren't HTTP parameters anymore they can't be defined here. The default is now the builtin loadgen defaults (by setting empty distributions). Missions that want easy-to-use presets can use `WithSmallLoadgenOptions` and `WithMediumLoadgenOptions` which are set to respect the network limits used in many of these tests. This change updates all of the soroban loadgen missions to use one of these presets based on the limits in the mission.